lib/datastax/graph/structure.js

```diff
@@ -1,144 +1,47 @@
 "use strict";
 
-const util = require("util");
+const utils = require("../utils");
 
 /**
- * @classdesc
- * Represents a graph Element.
- * @param id
- * @param label
- * @abstract
- * @memberOf module:datastax/graph
- * @constructor
+ * @deprecated
  */
 function Element(id, label) {
-  /**
-   * Gets the element id.
-   */
-  this.id = id;
-  /**
-   * Gets the element label.
-   * @type {String}
-   */
-  this.label = label;
+  throw new ReferenceError(utils.markAsNotSupported("class", "Element"));
```

Comment:
Style difference. Sometimes you use ``,`` and split it to 3 lines and sometimes you keep it in one line.

Reply:
This is actually Prettier thing, maybe this should be marked as an issue

Reply:
Yes, but sometimes you added ``,`` and than it splits into 3 lines, and when you don't it keeps it as one line

Reply:
I see... @Stapox35 Can you look into it?